.env.default

```diff
@@ -1,4 +1,4 @@
-DJANGO_ALLOWED_HOSTS=localhost
+DJANGO_ALLOWED_HOSTS=*
```

Comment:
```suggestion
DJANGO_ALLOWED_HOSTS=localhost,.gitpod.io
```
Also suggesting to keep the restriction and be explicit about other sources.

Reply:
I've committed the above without testing.

Reply:
Tested, it works 👍🏻